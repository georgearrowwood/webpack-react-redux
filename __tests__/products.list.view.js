import React from 'react';
import ProductsList from '../app/components/products/list';
import renderer from 'react-test-renderer';

let mockProductsList = [
  {id: 1, title: "test product 1"},
  {id: 2, title: "test product 2"},
]

test('Render products list', () => {
  const component = renderer.create(
    <ProductsList products={mockProductsList}/>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
