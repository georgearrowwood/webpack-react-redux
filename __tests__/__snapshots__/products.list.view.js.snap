// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render products list 1`] = `
<div>
  List:
  <ul>
    <li>
      1
       - 
      test product 1
       :
      <a
        href="#"
        onClick={[Function]}
      >
        X
      </a>
    </li>
    <li>
      2
       - 
      test product 2
       :
      <a
        href="#"
        onClick={[Function]}
      >
        X
      </a>
    </li>
  </ul>
</div>
`;
